### DISCUSSION TITLE
VEXCode JAR Support

### DISCUSSION THREAD

**Discussion by Xay123** [Jerrylum/path.jerryio#19] [Q&A]
So, I am using VEXCode with the [JAR template](https://github.com/JacksonAreaRobotics/JAR-Template) and was wondering is there an an easy way to port the output into this library which uses VEXCode and not PROS?

**Comment by Jerrylum** [comment]
I see. This editor can output files in any format you want. So, it is not only for PROS users. Now the question is, what do you want to output? For example, the autonomous code?
